docs/reference/pip_download.rst

```diff
@@ -46,8 +46,8 @@ Examples
 
   ::
 
-    $ pip download -d ./pip_downloads SomePackage
     $ pip download SomePackage  # equivalent to above
+    $ pip download -d . SomePackage
     $ pip download --no-index --find-links=/tmp/wheelhouse -d /tmp/otherwheelhouse SomePackage
```

Comment:
Those lines look like they are in the wrong order for the comment "equivalent to above"...